src/html/transfers.html

```diff
@@ -8,6 +8,18 @@ <h1>Your Transfers</h1>
     <!-- TODO: <nav> -->
   </header>
   <div data-behavior="transfers-container" style="margin-top: 3rem"></div>
+  <div data-behavior="deleteModal" class=modal style="display:none">
+    <div class="confirm-delete">
+      <img src="/img/delete-pink.svg" alt="">
```

Comment:
Not sure it's worth having two image files. This one seems to be the same as the other, just centered in a pink circle. That's pretty easy to pull off with CSS.

Reply:
The designed red bin has different colors though: #fc5b5b vs #dc1f25. The red is stronger on the pink background, and the red matches the other buttons on the white background.